src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3362,3 +3362,25 @@ def update_disk_encryption_set(instance, client, resource_group_name, key_url=No
     return instance
 
 # endregion
+
+
+# region Disk Access
+def create_disk_access(cmd, resource_group_name, disk_access_name, location, tags=None, no_wait=False):
```

Comment:
Can `location` be alternative ? We usually use resource group default location for a specific resource if location is not specified.